allennlp/predictors/predictor.py

```diff
@@ -294,17 +294,8 @@ def predict_batch_json(self, inputs: List[JsonDict]) -> List[JsonDict]:
     def predict_batch_instance(self, instances: List[Instance]) -> List[JsonDict]:
         for instance in instances:
             self._dataset_reader.apply_token_indexers(instance)
-        return [
-            sanitize(self._finalize_output(o)) for o in self._model.forward_on_instances(instances)
-        ]
-
-    def _finalize_output(self, output: JsonDict) -> JsonDict:
```

Comment:
Why are we not doing this anymore?

Reply:
I added this initially, then after doing some other refactoring I realized I really didn't need it, and instead the logic I put there should live in `Model.make_output_human_readable()`.